desktop/core/src/desktop/js/reactComponents/AlertComponent/AlertComponent.tsx

```diff
@@ -19,49 +19,89 @@ import Alert from 'cuix/dist/components/Alert/Alert';
 
 import huePubSub from 'utils/huePubSub';
 import './AlertComponent.scss';
+import { i18nReact } from '../../utils/i18nReact';
 
-interface ErrorAlert {
+interface HueAlert {
   message: string;
 }
 
+interface VisibleAlert {
+  alert: HueAlert;
+  type: 'error' | 'info' | 'warning';
+}
+
 const AlertComponent: React.FC = () => {
-  const [errors, setErrors] = useState<ErrorAlert[]>([]);
+  const [alert, setAlerts] = useState<VisibleAlert[]>([]);
 
-  useEffect(() => {
-    const hueSub = huePubSub.subscribe('hue.global.error', (newError: ErrorAlert) => {
-      if (!newError.message) {
-        return;
+  const updateAlerts = (alert, type) => {
+    if (!alert.message) {
+      return;
+    }
+    setAlerts(activeAlerts => {
+      // Prevent showing the same message multiple times.
+      // TODO: Consider showing a count in the error notification when the same message is reported multiple times.
+      if (activeAlerts.some(activeAlerts => activeAlerts.alert.message === alert.message)) {
+        return activeAlerts;
       }
+      return [...activeAlerts, { alert, type }];
+    });
+  };
 
-      setErrors(activeErrors => {
-        // Prevent showing the same message multiple times.
-        // TODO: Consider showing a count in the error notification when the same message is reported multiple times.
-        if (activeErrors.some(activeError => activeError.message === newError.message)) {
-          return activeErrors;
-        }
-        return [...activeErrors, newError];
-      });
+  useEffect(() => {
+    const hueSub = huePubSub.subscribe('hue.global.error', (newAlert: HueAlert) => {
+      updateAlerts(newAlert, 'error');
     });
     return () => {
       hueSub.remove();
     };
   }, []);
 
-  const handleClose = (errorObjToClose: ErrorAlert) => {
-    const filteredErrors = errors.filter(errorObj => errorObj !== errorObjToClose);
-    setErrors(filteredErrors);
+  useEffect(() => {
+    const hueSub = huePubSub.subscribe('hue.global.info', (newAlert: HueAlert) => {
+      updateAlerts(newAlert, 'info');
+    });
+    return () => {
+      hueSub.remove();
+    };
+  }, []);
+
+  useEffect(() => {
+    const hueSub = huePubSub.subscribe('hue.global.warning', (newAlert: HueAlert) => {
+      updateAlerts(newAlert, 'warning');
+    });
+    return () => {
+      hueSub.remove();
+    };
+  }, []);
```

Comment:
This is opinionated feedback :-) The functionality is logically closely related and the dependency array is the same so I think we should just use a single useEffect and improve the readability here.

You could even write something like:

```
function subscribeToAlert(type: string) {
  return huePubSub.subscribe(`hue.global.${type}`, (newAlert: HueAlert) => {
    updateAlerts(newAlert, type);
  });
}

useEffect(() => {
  const errorSub = subscribeToAlert('error');
  const infoSub = subscribeToAlert('info');
  const warningSub = subscribeToAlert('warning');

  return () => {
    errorSub.remove();
    infoSub.remove();
    warningSub.remove();
  };
}, []);
```